assets/src/edit-story/components/popup/index.js

```diff
@@ -29,9 +34,37 @@ const Container = styled.div.attrs(({ x, y }) => ({
 }))`
   position: fixed;
   z-index: 2147483646;
-  width: ${({ width }) => width}px;
+  width: ${({ width }) => (width === 'auto' ? width : `${width}px`)};
```

Comment:
We shouldn't allow `auto` here without some sort of `max-width`.

Reply:
Removed `auto` from popup and used default scrollbar width instead.

Reply:
That's not a scrollbar width though, right? Shouldn't this (at least optimally) try to follow the anchor's width?